rasa/core/policies/sklearn_policy.py

```diff
@@ -321,10 +323,15 @@ def load(cls, path: Union[Text, Path]) -> Policy:
         meta = json.loads(rasa.shared.utils.io.read_file(meta_file))
         zero_state_features = io_utils.pickle_load(zero_features_filename)
 
+        data = {"should_finetune": kwargs.get("should_finetune", False)}
```

Comment:
we can access the `should_fine` directly, no?

Reply:
(same for the others)

Reply:
We are purposefully not passing `should_finetune` in as a kwarg if it is not `True` so as not to break old policies.

Reply:
Mhm, but that's the specific `load` impl of the `SklearnPolicy`. I don't see what would break here?

Reply:
oh you mean making "should_finetune" a kwarg in the method definition?

Reply:
Yes